src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -129,12 +129,12 @@ public function createListFromChoices(iterable $choices, $value = null/*, $filte
      *
      * @param ChoiceLoaderInterface|Cache\ChoiceLoader $loader The loader or static loader to load
      *                                                         the choices lazily
-     * @param callable|Cache\ChoiceValue|null          $value  The callable or static option for
+     * @param callable|string|Cache\ChoiceValue|null   $value  The callable or static option for
      *                                                         generating the choice values
-     * @param callable|Cache\ChoiceFilter|null         $filter The callable or static option for
+     * @param callable|string|Cache\ChoiceFilter|null  $filter The callable or static option for
      *                                                         filtering the choices
      */
-    public function createListFromLoader(ChoiceLoaderInterface $loader, $value = null/*, $filter = null*/)
+    public function createListFromLoader($loader, $value = null/*, $filter = null*/)
```

Comment:
@HeahDude WDYT of this change? Aren't we missing a test case? Can you please send one if yes :pray:?

Reply:
We can keep the type-hint. `Symfony\Component\Form\ChoiceList\Factory\Cache\ChoiceLoader` also implements the `ChoiceLoaderInterface`.